app/move/steps/create.js

```diff
@@ -137,9 +150,14 @@ module.exports = {
       'gender_additional_information',
     ],
   },
-  '/move-date': {
-    pageTitle: 'moves::steps.move_date.heading',
+  '/move-date-range': {
     fields: ['date_from', 'date_to'],
+    ...moveDateStep,
+  },
+  '/move-date': {
+    controller: MoveDate,
+    fields: ['date', 'date_type', 'date_custom'],
+    ...moveDateStep,
```

Comment:
Same here - probably better to have the spread as the first item, that way you guarantee the properties here will override any defaults set in the shared object.

Reply:
Same as the `move-date-range` step, `...moveDateStep` should be the first property in the object.